### DISCUSSION TITLE
Where to get literature or information, or maybe someone will help you spend the night, to create a bot in a cart, see the description

### DISCUSSION THREAD

**Discussion by WhatIsLove1234** [community/community#29523] [Apps, API and Webhooks]
We need a bot that will enter text (link) on the site, receive a video, download it and send it to the person who sent the request. Thank you in advance ✨

**Comment by queenofcorgis** [comment]
Hi @WhatIsLove1234 , we're happy you're here 😀

However, we currently do not allow job listings or requests like this in Community on Discussions. It is something that we've added to the list to investigate if we can do so without overwhelming other conversations. Keep checking back as the Community grows and we add more capabilities!
